### ISSUE TITLE
wrong h3 indentation after bullet points

### ISSUE THREAD

**Issue by wischi-chr** [simov/markdown-viewer#49]
The following markdown is not rendered correctly

```
# Indentation Test

### Header 1
  - Bulletpoint 1
### Header 2 <---
  - Bulletpoint 2
```

On github it looks like this: https://gist.github.com/wischi-chr/dc48d7957e80fb4e2024f7f304489ced
![correct_markdown](https://i.imgur.com/dciJDNV.png)

but it's rendered like that:

![wrong_markdown](https://i.imgur.com/Qfi11b2.png)

**Comment by simov** [comment]
Thanks for the feedback! It's a compiler issue in both [Marked](https://github.com/chjj/marked) and [Remark](https://github.com/remarkjs/remark). You can open up an issue there and I'll update the dependencies here once the problem is resolved.

Otherwise just add a space between the header and the list items and it will work.

**Comment by wischi-chr** [comment]
I'm a bit lazy and on mobile right now so I copied the issue description to the compiler projects.

https://github.com/remarkjs/remark/issues/325

https://github.com/chjj/marked/issues/1047

**Comment by joshbruce** [comment]
Response from Marked so far:

Believe this is related to new lines and possible regex - believe you would see the same behavior from the original Daring Fireball implementation as well. Can you try the following:

```
### Header 1

- Bulletpoint 1

### Header 2

- Bulletpoint 2
```

**Comment by simov** [comment]
@wischi-chr it seems to be fixed in the latest release of `Remark`. I've published it with v3.6 of the extension.
